src/main/java/duke/DateTimeProcessor.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
```

Comment:
Consider separating imports into groups, and in lexicological order.

Reply:
Doing either will fail gradle's checkstyle. 😢